taxonomies/labels.txt

```diff
@@ -1684,7 +1684,7 @@ pl:Wzbogacane witaminami
 pt:Enriquecido com vitaminas
 sv:Vitaminberikat, Vitaminberikad
 
-en:Enriched wih minerals
+en:Enriched with minerals, Enriched wih minerals
```

Comment:
I wanted to keep it because I thought it works similar to categories. https://world.openfoodfacts.org/label/Enriched%20wih%20minerals has 30products so if you take out synonim "Enriched wih minerals" you lose those products, no ?

Reply:
@DanieliusAsm that's correct, we will need to remember to take them out later

Reply:
Good to know. You can leave it as it is and resolve the discussion